lib/dataContract/validateDataContractFactory.js

```diff
@@ -33,6 +39,18 @@ module.exports = function validateDataContractFactory(validator) {
     // TODO: Use validateSchema
     //  https://github.com/epoberezkin/ajv#validateschemaobject-schema---boolean
 
+    const serializedDataContract = encode(rawDataContract);
+    const serializedDataContractByteSize = Buffer.byteLength(serializedDataContract);
+
+    if (serializedDataContractByteSize > MAX_SERIALIZED_DATA_CONTRACT_BYTE_SIZE) {
+      return new ValidationResult([
```

Comment:
Let's just create only one instance of `ValidationResult` at the top and then just use `result.addError` and `result.merge`